calculate.rb

```diff
@@ -0,0 +1,58 @@
+class Calculate
```

Comment:
Class names should be nouns, as we are using them as the blueprint to make an object. i.e. it should be Calculator, not Calculate.